torch/_inductor/fx_passes/freezing_patterns.py

```diff
@@ -171,6 +170,32 @@ def addmm_fuse_replacement_second(inp, w1, w2, w3, b1, b2, b3):
     )
 
 
+@register_graph_pattern(
+    CallFunction(
```

Comment:
make a function to register pattern, and then invoke it once in freezing, and once in joint_graphs ?